gui/static/js/main.js

```diff
@@ -1,5 +1,7 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
+//var d3 = require('d3');
```

Comment:
Is this no longer necessary? In which case you should just remove them instead of keeping the comments. If you're going to comment out something and it's not meant to be permanent, then you should delete it or at least add a TODO to take care of it in the line above the comment.